tests/unit/contracts/test_forbidden.py

```diff
@@ -125,6 +125,24 @@ def test_is_invalid_when_forbidden_externals_but_graph_does_not_include_external
         ):
             contract.check(graph=graph)
 
+    def test_issue69(self):
```

Comment:
Thanks for these tests!

Only one thing - I'm not crazy about referring to Github issues in the tests - I think it's better to be explicit about what we're testing for here in the code. For example, this test could be called `test_ignore_imports_tolerates_duplicates`.

I'd be happy to change this myself unless you get there first!